### DISCUSSION TITLE
1.16.0

### DISCUSSION THREAD

**Discussion by th-ch** [pear-devs/pear-desktop#612] [General]
See [changelog](https://github.com/th-ch/youtube-music/blob/master/changelog.md#v1160) for the list of updates and the full diff.

Thanks to all contributors! 🏅 

<hr /><em>This discussion was created from the release <a href='https://github.com/th-ch/youtube-music/releases/tag/v1.16.0'>1.16.0</a>.</em>

**Comment by Bellisario** [comment]
Thanks to this new release (and obviously to @Araxeus and all contributors), the problem [here](https://github.com/th-ch/youtube-music/discussions/533#discussioncomment-1894174) seems to be resolved.
Very strange, but, better that!!!

**Comment by thijsi123** [comment]
It's been here for 9 hours?! I should've restarted it earlier😝.
Thanks and to all the contributors as well!

**Comment by Endermen1094** [comment]
Are there plans to add this to android

**Reply by Araxeus** [reply]
check out https://vancedapp.com/

**Reply by DereC4** [reply]
i can vouch; it's really reliable

**Comment by DereC4** [comment]
yay it's out!

**Comment by sanctifyy** [comment]
How do i install the update? Do i delete the previous version and just install the new one?

**Reply by Araxeus** [reply]
you can just install, no need to delete/uninstall previous versions

**Comment by CrazyLyte** [comment]
are any plan for ios? 
im new sorry in advance. Thanks you :)

**Reply by Araxeus** [reply]
This is a desktop app but check out https://github.com/qnblackcat/uYouPlus

**Comment by xdaWolf** [comment]
Hey, first of all, i absolutely love this app, thank you so much for the release^^ i wondered, if its possible to Fade between the songs? Will there be a plugin (if its possible)?
Thanks in advance!

**Reply by Araxeus** [reply]
you should request this feature in https://github.com/th-ch/youtube-music/issues
(more chances it will get implemented if its listed there)

**Comment by Hooooooman** [comment]
Is there any way to download this on fire tv?

**Comment by ercanzynk01** [comment]
hello how do i use this file? i'm new to this.

**Reply by Araxeus** [reply]
What's your os?

if its windows, download the exe installer
https://github.com/th-ch/youtube-music/releases/download/v1.16.0/YouTube-Music-Setup-1.16.0.exe

**Comment by BBMangoes** [comment]
Is there an equalizer? If not do you plan on adding one?

**Comment by mehdichaouch** [comment]
Hello @th-ch,

Very nice job 👏 (and also all contributors).

I found [ytm-wrapped](https://github.com/4v3ngR/ytm-wrapped) which is an Android wrapper app of YTMusic.
It's also a very young repository/project.

Do you think it would be possible, to do same (same workflow), I mean Android build, with your YTM which is really more advanced project?
